ldms/src/test/ldms-static-test-list.sh.in

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
```

Comment:
This does not need to be an execuable file with "#!/bin/sh" at the top to be included with "." in another script. I would like to see this not have those things, so it doesn't look to people like somethin they would run directly.